ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -262,16 +272,21 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{%     if router_bgp.vrfs is arista.avd.defined %}
 
-#### Router BGP EVPN VRFs
-{%     if router_bgp.vrfs is defined and router_bgp.vrfs is not none %}
+### Router BGP VRF Instances
 
-| VRF | Route-Distinguisher | Redistribute |
-| --- | ------------------- | ------------ |
+| VRF | Route-Distinguisher | Redistribute | EVPN Multicast |
+| --- | ------------------- | ------------ | -------------- |
 {%         for vrf in router_bgp.vrfs | arista.avd.natural_sort %}
 {%             set route_distinguisher = router_bgp.vrfs[vrf].rd | arista.avd.default("-") %}
 {%             set redistribute = router_bgp.vrfs[vrf].redistribute_routes | arista.avd.default("-") %}
-| {{ vrf }} | {{ route_distinguisher }} | {{ redistribute | join("<br>") }} |
+{%             if router_bgp.vrfs[vrf].evpn_multicast is arista.avd.defined(true) %}
+{%                 set evpn_multicast = "enabled" %}
+{%              else %}
```

Comment:
Indentation doesn't seem good with respect to above ```if```. Also correct the below endif indentation.